src/desktop/apps/auction/routes.js

```diff
@@ -25,11 +28,18 @@ export async function index(req, res, next) {
   const saleId = req.params.id
 
   try {
-    const { sale } = await metaphysics({
-      query: SaleQuery(saleId),
+    const { sale } = await MetaphysicsV2({
```

Comment:
is `MetaphysicsV2` a class? Why uppercase?

Reply:
We can totally call it `metaphysicsV2 `.